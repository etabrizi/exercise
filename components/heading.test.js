import * as React from 'react';
import Heading from './heading';
import renderer from 'react-test-renderer';

const Props = {
    title: "Pull up",
    infoToggle: true,
    searchToggle: true

}

describe('Heading component', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<Heading {...Props} />)
            .toJSON();
        expect(tree).toMatchSnapshot()
    });
});







