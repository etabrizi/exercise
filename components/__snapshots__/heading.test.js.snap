// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading component renders correctly 1`] = `
<div
  className="jsx-3648482963 show "
>
  <div
    className="Typist "
  >
    <span
      className="Cursor Cursor--blinking"
    >
      |
    </span>
  </div>
</div>
`;
